example_files/visualization/input_visualization_feature.html

```diff
@@ -483,13 +484,17 @@ <h2>Annual End Uses</h2>
                 y: endUseValue
               }]
             });
+
           });
-          $scope.annualEndUseChartOptions.chart['yDomain'] = [0, _.round(1.1 * annualTotalYMax)];
-        
+          if (annualTotalYMax > total_value_feature + 20000000) {
+          $scope.annualEndUseChartOptions.chart['yDomain'] = [0, _.round(total_value_feature)];
```

Comment:
For some reason the $scope.annualEndUseChartOptions.chart['yDomain'] sets the yDomain for all graphs and doesn't update for each feature based on whehter the condition is satisfied or not.